src/runtime_src/core/common/utils.cpp

```diff
@@ -26,6 +26,32 @@
 #include <limits>
 #include <boost/algorithm/string.hpp>
 
+std::string convert_version(std::stringstream& version)
```

Comment:
Convert version to what?  The name does not explain much and I can't follow the logic of the function.
```
return-type
function_name(...)
{
}
```
Looks like the function should be static and within unnamed namespace.

Reply:
This function  is  called  in src/runtime_src/core/common/info_platform.cpp .  So it  can't be static